src/odemis/driver/orsay.py

```diff
@@ -1146,8 +1047,7 @@ def __init__(self, name, role, parent, **kwargs):
         Defines the following VA's and links them to the callbacks from the Orsay server:
         • temperatureTarget: FloatContinuous, unit="°C", range=(-273.15, 10^3), setter=_setTemperatureTarget
         • temperature: FloatContinuous, readonly, unit="°C", range=(-273.15, 10^3)
-        • temperatureRegulation: EnumeratedVA, unit=None, choices={0: "off", 1: "on", 2: "rush"},
-                                 setter=_setTemperatureRegulation
+        • temperatureRegulation: BooleanVA, True: "on", False: "off", setter=_setTemperatureRegulation
```

Comment:
BTW, in docstrings, you normally shouldn't discuss about the internals. So no need to indicate the setters.